doc/rst/language/spec/modules.rst

```diff
@@ -273,9 +273,26 @@ Visibility Of A Module
 ~~~~~~~~~~~~~~~~~~~~~~
 
 A top-level module is available for use (:ref:`Using_Modules`) or
-import (:ref:`Importing_Modules`) anywhere. The visibility of a nested module is
-subject to the rules of :ref:`Visibility_Of_Symbols`, where the nested module is
-considered a "module-scope symbol" of its outer module.
+import (:ref:`Importing_Modules`) anywhere.
+
+For the purpose of ``use``, the visibility of a nested module is subject to the
+rules of :ref:`Visibility_Of_Symbols`, where the nested module is considered a
+"module-scope symbol" of its outer module.  If the module is currently in scope,
+then it may be used with just its name.  The module may alternatively be
+accessed explicitly with all the outer modules surrounding it to the top level,
+or relatively from the current scope with ``this`` or ``super`` components.
+Which of ``this`` or ``super`` is appropriate depends on its placement relative
+to the current scope - if the submodule is within the current scope, then
+``this`` is used; if the submodule is within a parent module's scope, then one
+or more ``super`` components can be used.
+
+For the purpose of ``import``, the module is known to be defined at the same
+location as for ``use`` statements.  However, in order to ``import`` it, either
```

Comment:
I honestly don't know what the 1st sentence here means.

Reply:
It's intended as a follow on to the first sentence of the previous paragraph.  Basically, the "scope" for the name is the same, the compiler knows it is a symbol in the enclosing module, even though you have to give path to it instead of naming it directly.  But I should make that more explicit or something